### PR TITLE
Minor changes:

### PR DESCRIPTION
**Pull request by vaishnavipaliya11**
styled components.

### CODE REVIEW

File: src/Components/Modal.jsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import { usePlay } from "../context/playListContext";
+import "./modal.css";
+import { useState } from "react";
+import { createPlaylist } from "../Utils/createPlaylist";
+import { addToPlaylist } from "../Utils/addToPlaylist";
```
Comment (shraddha15-creator) [inline src/Components/Modal.jsx:6]: you can import this from index.js file

File: src/Pages/Login/Login.jsx [hunk 2]
```diff
@@ -1,83 +1,100 @@
...
-          setAuth(true)
-          navigate("/")
-        console.log( response);
-        } catch (error) {
-          console.log(error);
-        }
-
-   };
+  const testHandler = async (e) => {
+    console.log("called");
```
Comment (shraddha15-creator) [inline src/Pages/Login/Login.jsx]: remove console

File: src/Pages/Login/Login.jsx [hunk 3]
```diff
@@ -1,83 +1,100 @@
...
+    try {
+      const { data } = await axios.post("api/auth/login", {
+        email: "adarshbalika@gmail.com",
+        password: "adarshBalika123",
+      });
+      localStorage.setItem("token", data.encodedToken);
+      setAuth(true);
+      navigate("/");
+    } catch (error) {
+      alert(error);
```
Comment (shraddha15-creator) [inline src/Pages/Login/Login.jsx:45]: use console.error instead of alert

File: src/Pages/Playlist/Playlist.jsx [hunk 4]
```diff
@@ -1,11 +1,75 @@
-import React from 'react'
+import React from "react";
+import { useEffect } from "react";
```
Comment (shraddha15-creator) [inline src/Pages/Playlist/Playlist.jsx]: import it like this --> ` import React, { useEffect } from 'react'; `

File: src/Pages/Playlist/Playlist.jsx [hunk 5]
```diff
@@ -1,11 +1,75 @@
...
+import axios from "axios";
+import { usePlay } from "../../context/playListContext";
+import { getUserToken } from "../../Utils/getUserToken";
+import { VideoCard } from "../../Components/VideoCard/VideoCard";
+import "./Playlist.css";
+import deletePlayList from "../../Utils/deletePlayList";
+import { useNavigate } from "react-router-dom";
```
Comment (shraddha15-creator) [inline src/Pages/Playlist/Playlist.jsx:8]: You can sort all the default imports at the top and then manual imports at the bottom. 
```
import React, { useEffect } from 'react';

 import axios from "axios";

import { useNavigate } from "react-router-dom";

import { usePlay } from "../../context/playListContext";

import { VideoCard } from "../../Components/VideoCard/VideoCard";

import { getUserToken } from "../../Utils/getUserToken";

import deletePlayList from "../../Utils/deletePlayList";

import "./Playlist.css";

```

File: src/Pages/Sign/Signup.jsx [hunk 6]
```diff
@@ -55,68 +55,64 @@ const Signup = () => {
...
               name="Password"
               onChange={handelUserInput}
               value={userData.Password}
             ></input>
           </div>
 
-          <div style={{color:"red"}}>
-          <p>{erorMsg && "Enter the feilds"}</p>
+          <div style={{ color: "red" }}>
+            <p>{erorMsg && "Enter the feilds"}</p>
```
Comment (shraddha15-creator) [inline src/Pages/Sign/Signup.jsx]: I think you have mistyped the _fields_ spelling here.

File: src/Pages/Sign/Signup.jsx [hunk 7]
```diff
@@ -55,68 +55,64 @@ const Signup = () => {
...
-          </Link>
-      
-            <button onClick={signUpHandler}>Sign up</button>
-            
+            <Link to="/login">
+              <p> Already have an account?</p>
+            </Link>
+
+            <button className="remove-card-btn" onClick={signUpHandler}>
+              Sign up
```
Comment (shraddha15-creator) [inline src/Pages/Sign/Signup.jsx:114]: if already have an account? then we need to *sign in* right?

File: src/Pages/SingleProduct/SingleProduct.jsx [hunk 8]
```diff
@@ -9,14 +9,15 @@ import { AiFillLike } from "react-icons/ai";
 import { useAuth } from "../../context/authContext";
 import { addToLike } from "../../Utils/addToLike";
 import { useLiked } from "../../context/likeContext";
+import { useWatchLater } from "../../context/watchContext";
+import { addToWatchLater } from "../../Utils/addToWatchLater";
```
Comment (shraddha15-creator) [inline src/Pages/SingleProduct/SingleProduct.jsx:13]: use index.js files in all the folder, and export all files there and then wherever you want to import it you can import it from there. 
Like this: 
```
import { useAuth, useLiked, useWatchLater } from "../../context";

import { addToLike, addToWatchLater } from "../../Utils"
```

this looks more clean

File: src/Utils/createPlaylist.js [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+        },
+  
+        {
+          headers: {
+            authorization: getUserToken(),
+          },
+        }
+      );
+  
+      console.log(response);
```
Comment (shraddha15-creator) [inline src/Utils/createPlaylist.js]: remove console

File: src/Utils/createPlaylist.js [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+      playListDispatch({
+        type: "CREATE_PLAYLIST",
+        payload: response.data.playlists,
+      });
+
+      setUserList({ ...userList, title: "" });
+    } catch (error) {
+      console.log(error);
```
Comment (shraddha15-creator) [inline src/Utils/createPlaylist.js]: use console.error

File: src/Utils/deletePlayList.js [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+const deletePlayList = async (_id, playListDispatch) => {
+  try {
+    const { data } = await axios({
+        method: "DELETE",
+        url: `/api/user/playlists/${_id}`,
+        headers: {
+          authorization: getUserToken(),
+        },
+      });
+      console.log("from func", data);
```
Comment (shraddha15-creator) [inline src/Utils/deletePlayList.js]: remove console

File: src/Utils/deletePlayList.js [hunk 12]
```diff
@@ -0,0 +1,25 @@
...
+    playListDispatch({
+        type: "DELETE_FROM_PLAYLIST",
+        payload:data.playlists,
+      })
+   
+  } catch (error) {
+    console.log(error);
```
Comment (shraddha15-creator) [inline src/Utils/deletePlayList.js]: here as well

**Review by shraddha15-creator — APPROVED** [review]
Suggested some changes. 
Rest looks good

File: src/Pages/Login/Login.jsx [hunk 13]
```diff
@@ -1,83 +1,100 @@
...
+      const response = await axios.post(`/api/auth/login`, {
+        email: userDetails.email,
+        password: userDetails.password,
+      });
+      localStorage.setItem("token", response.data.encodedToken);
+      setAuth(true);
+      navigate("/");
+      console.log(response);
+    } catch (error) {
+      console.log(error);
```
Comment (znjs) [inline src/Pages/Login/Login.jsx]: can make it `console.error` for sematic meaning

File: src/Pages/Login/Login.jsx [hunk 14]
```diff
@@ -1,83 +1,100 @@
...
+    e.preventDefault();
+    try {
+      const response = await axios.post(`/api/auth/login`, {
+        email: userDetails.email,
+        password: userDetails.password,
+      });
+      localStorage.setItem("token", response.data.encodedToken);
+      setAuth(true);
+      navigate("/");
+      console.log(response);
```
Comment (znjs) [inline src/Pages/Login/Login.jsx]: can remove the `console.log`

File: src/Pages/Playlist/Playlist.jsx [hunk 15]
```diff
@@ -1,11 +1,75 @@
...
+  useEffect(() => {
+    (async () => {
+      try {
+        const response = await axios.get("/api/user/playlists", {
+          headers: {
+            authorization: getUserToken(),
+          },
+        });
+
+        console.log("response", response.data.playlists);
```
Comment (znjs) [inline src/Pages/Playlist/Playlist.jsx]: can remove the `console.log`

File: src/Pages/Watch/Watch.jsx [hunk 16]
```diff
@@ -3,23 +3,32 @@ import { useWatchLater } from "../../context/watchContext";
...
   const { watchLaterState } = useWatchLater();
   const { watchLater } = watchLaterState;
   const { watchLaterDispatch } = useWatchLater();
 
   return (
-    <div className="watchLater-container">
-      <h2>This is Watch later page.</h2>
+    <div className="page-cards-wrapper">
 
+    {watchLater.length === 0 ?
```
Comment (znjs) [inline src/Pages/Watch/Watch.jsx:17]: remove ===0 and try `!watchLater.length`

File: src/Utils/addDataToList.js [hunk 17]
```diff
@@ -0,0 +1,6 @@
+const addDataToList = (video,playListDispatch)=>{
+    playListDispatch({type:"SELECTED_ID",payload:video})
+    playListDispatch({type:"MODAL",payload:true})
```
Comment (znjs) [inline src/Utils/addDataToList.js:3]: can use variables instead of string to avoid typo bugs

File: src/Utils/addToPlaylist.js [hunk 18]
```diff
@@ -0,0 +1,25 @@
...
+          authorization: getUserToken(),
+        },
+      }
+    );
+    playListDispatch({
+      type: "PLAYLIST_VIDEO",
+      payload: response.data.playlist,
+    });
+  } catch (error) {
+    console.log(error);
```
Comment (znjs) [inline src/Utils/addToPlaylist.js]: can use `console.error` for sematic meaning

**Review by znjs — APPROVED** [review]

**Review by simran61 — APPROVED** [review]

File: src/Pages/Sign/Signup.jsx (hunk 7 above)
Comment (vaishnavipaliya11) [reply src/Pages/Sign/Signup.jsx:114]: user can click to "already have account and it will be redirect to login page"

**Pull request merged**